src/account/Settings.tsx

```diff
@@ -568,6 +594,11 @@ const styles = StyleSheet.create({
   bottomSheetButton: {
     marginTop: Spacing.Regular16,
   },
+  value: {
+    ...fontStyles.regular,
```

Comment:
typeScale instead of fontStyles.

Reply:
I wouldn't want to change just this one without updating all of the settings menu, which is all still on `fontStyles`. Also the figma designs themselves haven't been updated to the new typeScale so i think it is best to wait